code/datums/traits/negative.dm

```diff
@@ -479,24 +479,27 @@
 	desc = "Talking to people is very difficult for you, and you often stutter or even lock up."
 	value = -1
 	gain_text = "<span class='danger'>You start worrying about what you're saying.</span>"
-	lose_text = "<span class='notice'>You feel easier about talking again.</span>" //if only it were that easy!
+	lose_text = "<span class='notice'>You feel more comftorable with talking again.</span>" //if only it were that easy! (second this - etherware)
 	medical_record_text = "Patient is usually anxious in social encounters and prefers to avoid them."
+	process = TRUE
 	var/dumb_thing = TRUE
 
 /datum/quirk/social_anxiety/on_process(delta_time)
 	var/nearby_people = 0
-	for(var/mob/living/carbon/human/H in oview(3, quirk_holder))
-		if(H.client)
```

Comment:
don't remove this client check, it makes sure the people around you are actually players

Reply:
oh oops! i actually removed that so i could test it locally